apps/core/scenes/search/SortTabsWrapper.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { SortTabs } from '@kiwicom/margarita-components';
+import { StyleSheet, designTokens } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import {
+  withSearchContext,
+  type SearchContextState,
+  type SortTypes,
+} from './SearchContext';
+
+type Props = {
+  sort: SortTypes,
+  setSortBy: SortTypes => void,
```

Comment:
Please use extact type, and read-only props

Reply:
Done!